step-capstone/src/components/Flight.js

```diff
@@ -0,0 +1,26 @@
+import React from 'react'
+import { Box, Grid, Typography } from '@material-ui/core'
+
+/*
+Takes data in this form:
+        {
+          finalized: true,
+          type: "flight",
+          departureAirport: "BOS",
+          arrivalAirport: "SFO",
+          departureDate: "4:00pm EST",
```

Comment:
this would also include the date (may 17th for example) correct?

Reply:
Hmm I was thinking since travel objects in the itinerary would only show up under that specific day we wouldn't need to display the date.

Reply:
Remember that flights might depart and arrive on different dates, so you would need a way to note that if you don't store the departure and arrival date.